### PR TITLE
PasswordDialog: add ability to toggle text echo mode

### PR DESCRIPTION
**Pull request by mmbyday**

### CODE REVIEW

**Comment by MaxXor** [comment]
Yes, an eye icon would be better.

File: components/PasswordDialog.qml [hunk 1]
```diff
@@ -136,12 +137,33 @@ Item {
...
+                            anchors.fill: parent
+                            cursorShape: Qt.PointingHandCursor
+                            hoverEnabled: true
+                            onClicked: {
+                                passwordInput.echoMode = isHidden ? TextInput.Normal : TextInput.Password;
+                                isHidden = !isHidden;
+                            }
+                            onEntered: {
+                                parent.opacity = 0.9
+                                parent.width = 28
```
Comment (xmrdsc) [inline components/PasswordDialog.qml]: All integer constants should terminate with `* scaleRatio`.

Comment (mmbyday) [reply components/PasswordDialog.qml]: Updated. Thank you.

**Comment by selsta** [comment]
Do you remember where you found the eye icon? I’m looking for a 2x version of it.

**Pull request merged**